pom.xml

```diff
@@ -567,8 +567,8 @@
         <version>1.1.0</version>
         <configuration>
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.1-SNAPSHOT.0,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline[6.2.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```

Comment:
Does this mean that until we release 6.1.0, we won't be able to build and run plugin in develop?

Reply:
I think we can modify it to 6.2.0-SNAPSHOT if we add new apis in 6.2.0-SNAPSHOT. Changing it to 6.2.0-SNAPSHOT